### ISSUE TITLE
Broken links in Elmish page

### ISSUE THREAD

**Issue by filippo** [fable-elmish/fable-elmish.github.io#31]
Hi, there are some broken links in the Elmish page:
https://elmish.github.io/elmish/

There's a /docs/ missing.

Here are the links:

"basics example"
https://elmish.github.io/elmish/basics.html
Should be:
https://elmish.github.io/elmish/docs/basics.html

Parent-child example:
https://elmish.github.io/elmish/parent-child.html
Should be:
https://elmish.github.io/elmish/docs/parent-child.html

Subscriptions example:
https://elmish.github.io/elmish/subscriptions.html
Should be:
https://elmish.github.io/elmish/docs/subscription.html

If you want I can try to make a pull request

**Comment by et1975** [comment]
I thinks these were links to actual minimal apps. The apps are still there
I believe, just have to check previous version of the docs to see where.

**Comment by MangelMaxime** [comment]
Ah yes, I removed the "embedded" version of the demos when re-designing https://elmish.github.io/ website.

Instead of creating an embedded version of the example, I directly linked to the demo websites. See links under the **Examples** category at the bottom of https://elmish.github.io/.

I see 2 solutions:
1. We fix the broken links to point to the demo app or repository
2. We re-add embedded version of the demos application

The problem I have with the current demo pages is that they aren't really friendly... There is no explanation, no links to the source code, etc.

Should be add a documentation website to the demo or should we just link to the repository and have a well written README.md file ?

If you add a documentation website, it is possible to use literate documentation to have the code and the documentation in the same file.

**Comment by et1975** [comment]
Right, the live samples were kinda dangling in vacuum. I'm onboard with the 1st option, having links and better readme that lets you see it live.
